ci/terraform/oidc/authdev1.tfvars

```diff
@@ -14,6 +14,7 @@ account_intervention_service_call_enabled   = true
 account_intervention_service_action_enabled = true
 account_intervention_service_abort_on_error = true
 send_storage_token_to_ipv_enabled           = true
+call_ticf_cri                               = false
```

Comment:
Do we want to set this to true? (I'd argue for yes to make all dev envs consistent)